src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+
+
+class ElectronicKineticEnergy(QuantumEnergy):
+    """
+    Physical property section describing the electronic kinetic energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class XCEnergy(QuantumEnergy):
+    """
+    Physical property section describing the exchange-correlation (XC) energy of a (sub)system,
+    calculated using the functional stored in XC_functional.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class XCPotentialEnergy(QuantumEnergy):
+    """
+    Physical property section describing the potential energy contribution to the exchange-correlation (XC) energy,
+    i.e., the integral of the first order derivative of the functional
+        stored in XC_functional (integral of v_xc*electron_density), i.e., the component
+        of XC that is in the sum of the eigenvalues. Value associated with the
+        configuration, should be the most converged value.          |
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+# ? XCCorrelationEnergy?
+class CorrelationEnergy(QuantumEnergy):
+    """
+    Physical property section describing the correlation energy of a (sub)system,
+    calculated using the method described in XC_functional.
+    """
+
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule',
+        description="""
+        The value of the correlation energy.
+        """,
+    )
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+# ? XCExchangeEnergy?
+class ExchangeEnergy(QuantumEnergy):
+    """
+    Physical property section describing the exchange energy of a (sub)system,
+    calculated using the method described in XC_functional.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ZeroTemperatureEnergy(QuantumEnergy):
+    """
+    Physical property section describing the total energy of a (sub)system extrapolated to $T=0$, based on a free-electron gas argument.
+    """
+    # ! Someone check this description!
+    # ? Do we really want to specify the method here? This can't be user-defined?
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
I think this should be introduced in the context of a workflow. Would remove it here.